scripts/sem_bleaching_map.py

```diff
@@ -571,37 +917,72 @@ def main(args):
+                        help="SEM dwell time in s, , for taking the SEM image after "
+                             "bleaching procedure")
     parser.add_argument("--pxs", "-p", dest="pxs", type=float, required=True,
                         help="distance between 2 spots in nm")
     parser.add_argument("--subpx", "-s", dest="subpx", type=int, default=1,
                         help="number of sub-pixels scanned by the ebeam for "
-                            "each pixel acquired by the CCD. Must be a the "
-                            "square of an integer.")
+                             "each pixel acquired by the CCD. Must be a the "
+                             "square of an integer.")
+    parser.add_argument("--gridpitch", "-g", dest="gridpitch", type=int, default=0,
+                        help="pitch for gridscanning in number of SR pixels."
+                             "Setting pitch to 0 disables gridscanning.")
     parser.add_argument("--roi", dest="roi", required=True,
                         help="e-beam ROI positions (ltrb, relative to the SEM "
                              "field of view)")
     parser.add_argument("--lpower", "-lp", dest="lpower", type=float, default=0.02,
                         help="excitation light power on the first fluorescence pixel. "
-                            "Choose the value between 0.004 and 0.4 (in W). "
-                            "Power will be gradually scaled up to the maximum (0.4 W) "
-                            "on the last pixel to compensate photobleaching effect "
-                            "in the signal drop.")
+                             "Choose the value between 0.004 and 0.4 (in W).")
+    parser.add_argument("--lemissions", "-le", dest="lemissions", type=int, default=0,
+                        help="emissions source (numbered from 0 to n-1) used "
+                             "for the SR pixels.")
+    parser.add_argument("--exptime", "-e", dest="exptime", type=float, default=2.0,
+                        help="exposure time used "
+                             "for the SR pixels (in seconds).")
+    parser.add_argument("--overviewlightpower", "-olp", dest="olpower", type=float, nargs="+", default=[],
+                        help="excitation light power used for overview image"
+                             "that is acquired before and after the scan.")
+    parser.add_argument("--overviewlightemissions", "-ole", dest="olemissions", type=int, nargs="+", default=[],
+                        help="emissions source (numbered from 0 to n-1) used "
+                             "for overview image that is acquired before and "
+                             "after the scan.")
+    parser.add_argument("--overviewexposuretime", "-oet", dest="oexptime", type=float, nargs="+", default=[],
+                        help="exposure time that is used for overview image"
+                             "that is acquired before and after the scan.")
```

Comment:
As I mentioned above these parameters should be lists not float. Please make this is correct so there'd not be confusion between them and the 3 above.